src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle extends Rectangle {
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color, firstLeg, secondLeg, "triangle");
```

Comment:
Passing a string literal like 'triangle' directly into the superclass constructor is not ideal as it's a magic string. Consider defining it as a constant.